exe/wallet/Main.hs

```diff
@@ -182,6 +192,18 @@ exec manager args
         wId <- args `parseArg` longOption "id"
         runClient $ void $ deleteWallet (ApiT wId)
 
+    | args `isPresent` longOption "version" = do
+        let cabal = B8.unpack $(embedFile "cardano-wallet.cabal")
```

Comment:
we're not shipping `cardano-wallet.cabal`, so it wouldn't work, would it? :/

Reply:
It does, the file is embedded at compile-time. This won't even compile without the file.